bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -9,8 +9,23 @@
 ################################################################################
 
 from ROOT import pythonization
+from cppyy.gbl.ROOT import RDF
```

Comment:
The problem of having this import here is that, for any user application, we will import `RDF`, always. In order to avoid that, the import should be protected by the `if` in `pythonize_rdataframe`.